geoengine/workflow.py

```diff
@@ -7,11 +7,12 @@
 from __future__ import annotations
 
```

Comment:
In The notebook, you state that you would supress a warning that occurs when `clip_to_query_rectangle=True`, but you don't clip.

Reply:
3e3b8f41d05f5080deef0bdb57bdc5f110448b51